draft-packages/modal/KaizenDraft/Modal/Primitives/ModalHeader.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from "react"
 
-import { IconButton } from "@kaizen/component-library"
+import { IconButton } from "@kaizen/draft-button"
```

Comment:
Cluny primary button?

Reply:
Descoped

Reply:
Rescoped after discussing with Max + Doug from the a11y capture project